Data/Sys/GameSettings/RSB.ini

```diff
@@ -19,3 +19,6 @@ EmulationIssues = Classic mode score report needs real xfb. Nes masterpieces and
 
 [Video_Stereoscopy]
 StereoConvergence = 136
+
+[Video_Hacks]
+EFBAccessEnable = False # Fix online FPS
```

Comment:
This sounds like a horrible hack. Just get a fast enough computer to still support EFB access ;)